qa/rpc-tests/wallet_listunspent.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016-2022 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    get_coinbase_address,
+    nuparams,
+    start_nodes,
+    wait_and_assert_operationid_status,
+    NU5_BRANCH_ID
+)
+
+from decimal import Decimal
+
+def unspent_total(unspent):
+    total = 0
+    for item in unspent:
+        total += item['amount']
+    return total
+
+class WalletListUnspent(BitcoinTestFramework):
+    def setup_nodes(self):
+        return start_nodes(4, self.options.tmpdir, [[
+            nuparams(NU5_BRANCH_ID, 201),
+        ]] * 4)
+
+    def run_test(self):
+        assert_equal(self.nodes[0].getbalance(), 250)
+        assert_equal(self.nodes[1].getbalance(), 250)
+
+        # Activate NU5
+        self.nodes[1].generate(1) # height 201
+        self.sync_all()
+        assert_equal(self.nodes[0].getbalance(), 260) # additional 10 ZEC matured
```

Comment:
Consider defining a `matured_at_height(height)` helper function and specifying the offset from that.

Reply:
What I meant was just:
```
def expected_matured_at_height(height):
    return (height-200)*10 + 250
```